tmt/steps/provision/vagrant.py

```diff
@@ -225,7 +225,7 @@ def how_libvirt(self):
         try:
             self.add_provider('libvirt', 'qemu_use_session = true')
         except GeneralError as error:
-            self.debug(error)
+            self.debug(str(error))
```

Comment:
I'll fix it in other way, the way it was meant: https://github.com/psss/tmt/blob/fb8cfe0c65c8661a90cf09a4e4f469340ffbdceb/tmt/steps/provision/vagrant.py#L426

I'll open a PR today.

Reply:
```
=> if type(val) is not str:
```